data/data/azure/bootstrap/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "bootstrap_ip" {
```

Comment:
is this used?

Reply:
I see now from https://github.com/openshift/installer/pull/5010/commits/9cf8369d7d65405d6d7574b77d377f5616676647 This is indeed used for `gather`.